es6-promises/main.js

```diff
@@ -0,0 +1,9 @@
+import takeAChance from './take-a-chance.js';
+
+const chanceTaken = takeAChance('Brenda');
+
+chanceTaken
+  .then(
+    (value) => console.log(value),
+    (error) => console.error(error.message)
+  );
```

Comment:
This is OK but it's best to use `.catch` instead of having both fulfill and reject handlers in `.then`.